src/data-publication/reports/tables/Aggregate2.jsx

```diff
@@ -46,95 +84,163 @@ const Aggregate2 = React.forwardRef((props, ref) => {
     return 0
   })
 
+  return fixUnaggregatedValue(sorted)
+}
+
+export const buildCSVRowsAggregate2 = (report) => {
```

Comment:
I believe it would make sense to move the `buildCSVRowsAggregate2` function to a utility file since it isn't being used in this file.

What do you think?

Reply:
Done.